benchmark.py

```diff
@@ -226,22 +286,49 @@ def main():
   parser.add_argument("--sleep_between_retry",
                       help="number of seconds to sleep between each retry.",
                       type=int, default=5)
-
-  create_del_parser = parser.add_mutually_exclusive_group(required=False)
-  create_del_parser.add_argument("--create_delete", dest="create_delete",
-                                 action="store_true",
-                                 help="if you want to create/delete new VM.")
-  create_del_parser.add_argument("--no-create_delete", dest="create_delete",
-                                 action="store_false")
-  parser.set_defaults(create_delete=True)
-
-  skip_setup_parser = parser.add_mutually_exclusive_group(required=False)
-  skip_setup_parser.add_argument("--setup", dest="setup",
-                                 action="store_true",
-                                 help="if you want to run setup.")
-  skip_setup_parser.add_argument("--no-setup", dest="setup",
-                                 action="store_false")
-  parser.set_defaults(setup=True)
+  parser.add_argument("--db_instance_name",
+                      help="the name of the gcloud instance",
+                      default="envoy-db-instance")
+  parser.add_argument("--tier", help="the tier of GCloud SQL service",
+                      default="db-n1-standard-2")
+  parser.add_argument("--db_username", help="username on the DB",
+                      default="root")
+  parser.add_argument("--table_name", help=("the table which stores "
+                                            "the benchmarking data"),
+                      default="envoy_stat")
+  parser.add_argument("--envoy_hash",
+                      help="the hash of envoy version",
+                      required=True)
+  parser.add_argument("--runid",
+                      help="the run id of this benchmark",
+                      default="0")
+  parser.add_argument("--database", help="name of the database",
+                      default="envoy_stat_db")
+
+  utils.CreateBooleanArgument(parser, "create_delete",
+                              ("if you want to create/"
+                               "delete new benchmarking VM."),
+                              create_delete=True)
+
+  utils.CreateBooleanArgument(parser, "setup",
+                              ("if you want to run"
+                               " setup on benchmarking VM."),
+                              setup=True)
+
+  utils.CreateBooleanArgument(parser, "create_db_instance",
+                              ("turn on if you want to create"
+                               " a Google Cloud SQL instance"),
+                              create_db_instance=True)
+
+  utils.CreateBooleanArgument(parser, "create_db",
+                              ("turn on if you want"
+                               " to create the DB"),
+                              create_db=True)
+
+  utils.CreateBooleanArgument(parser, "delete_db",
```

Comment:
I would remove this from the main benchmark utility. There should maybe be an independent utility for this, but it doesn't make sense in the context of the benchmark.